src/cmd/sql_cmd.h

```diff
@@ -349,8 +349,13 @@ void PrintProcedureInfo(const hybridse::sdk::ProcedureInfo& sp_info) {
     if (sp_info.GetType() == hybridse::sdk::kReqDeployment) {
         type_name = "Deployment";
     }
+    std::string sql = sp_info.GetSql();
+    if (sp_info.GetType() == hybridse::sdk::kReqDeployment) {
+        // replace the sql "CREATE PROCEDURE" with "DEPLOY"
+        sql.replace(0u, 16u, "DEPLOY");
```

Comment:
Will users use "CREATE PROCEDURE"? Replace string directly is strange if we support standard "CREATE PROCEDURE" in the future.

Reply:
Actually, We hope user use Deployment instead of Procedure, and we should decouple the code between them. But right now sql haven't equal Deployment and Procedure, so I just replace string directly

Reply:
> Will users use "CREATE PROCEDURE"? Replace string directly is strange if we support standard "CREATE PROCEDURE" in the future.

Actually, We hope user use Deployment instead of Procedure, and we should decouple the code between them. But right now sql haven't equal Deployment and Procedure, so I just replace string directly